File: gluecodium/src/test/resources/smoke/enums/output/android/com/example/smoke/EnumWithAlias.java
/*

 *
 */

package com.example.smoke;


public enum EnumWithAlias {
    ONE(2),
    TWO(3),
    THREE(4);

    public final static EnumWithAlias FIRST = EnumWithAlias.ONE;
    public final static EnumWithAlias THE_BEST = EnumWithAlias.FIRST;

    /**
     * @hidden
     */
    public final int value;

    EnumWithAlias(final int value) {
        this.value = value;
    }
}

File: gluecodium/src/test/resources/smoke/enums/output/android-kotlin/com/example/smoke/EnumWithAlias.kt
/*

 *
 */

@file:JvmName("EnumWithAlias")

package com.example.smoke


enum class EnumWithAlias(@JvmField val value: Int) {
    ONE(2),
    TWO(3),
    THREE(4);

    companion object {
        @JvmField val FIRST = EnumWithAlias.ONE
        @JvmField val THE_BEST = EnumWithAlias.FIRST
    }

}
